src/solver/LSSOLLeastSquareSolver.cpp

```diff
@@ -35,6 +35,23 @@ LSSOLLeastSquareSolver::LSSOLLeastSquareSolver(const LSSOLLSSolverOptions & opti
   TVM_PROCESS_OPTION(printLevel, fp_)
   TVM_PROCESS_OPTION(rankTol, fp_)
   TVM_PROCESS_OPTION(warm, fp_)
+  ls_.forceMinSize(false);
```

Comment:
Is `TVM` the proper place to define these default values? Versus defining them in `mc_rtc`?
I don't have strong opinion on this, but it's worth considering the question as we impose these values to all `TVM` uses.

Reply:
The issue here is that mc_rtc uses the default ls solver so it doesn't know what options to pass. That would also prevent drop-in replacement when switching the default solver in TVM